### ISSUE TITLE
Warning each batch: Attempting to cast a BatchEncoding to type torch.float32. This is not supported.

### ISSUE THREAD

**Issue by mgwillia** [TXH-mercury/COSA#3]
As the title says, when I run zero-shot inference with, for example, ActivityNet, from pre-trained weights, I get this warning once per batch. The final retrieval results seem plausible, so I'm wondering if this warning message is to be expected, or if it indicates some broken environment on my end.

**Comment by TXH-mercury** [comment]
Hi@mgwillia , If the evaluation metric are normal, then I think this warning could be ignored.
